sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/webpack-assets/prod.config.js

```diff
@@ -1,22 +1,21 @@
 const ScalaJS = require("./scalajs.webpack.config");
-const Merge = require("webpack-merge");
+const { merge } = require("webpack-merge");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 const path = require("path");
 const rootDir = path.resolve(__dirname, "../../../..");
 const resourcesDir = path.resolve(rootDir, "src/main/resources");
 
-const WebApp = Merge(ScalaJS, {
-  mode: "production",
+const WebApp = merge(ScalaJS, {
   entry: {
     app: [path.resolve(resourcesDir, "./entry.js")]
   },
   module: {
     rules: [
       {
         test: /\.css$/,
-        use: ["style-loader", MiniCssExtractPlugin.loader, "css-loader"]
+        use: [MiniCssExtractPlugin.loader, "css-loader"]
```

Comment:
`style-loader` should not be included as `html-webpack-plugin` will automatically handle the CSS files